test/test_translator.py

```diff
@@ -195,110 +354,253 @@ def test_get_road_direction():
 
-def test_get_event_status():
+#--------------------------------------------------------------------------------Unit test for get_event_status function--------------------------------------------------------------------------------
+def test_get_event_status_active():
     test_starttime_string = "2020-12-16T08:45:03Z"
     test_endtime_string=''
     test_event_status=icone_translator.get_event_status(test_starttime_string,test_endtime_string)
     valid_event_status= "active"
-
     assert  test_event_status==valid_event_status
 
+def test_get_event_status_planned():
     test_starttime_string = "2030-12-16T08:45:03Z"
     test_endtime_string = ''
     test_event_status = icone_translator.get_event_status(test_starttime_string, test_endtime_string)
     valid_event_status = "planned"
-
     assert test_event_status == valid_event_status
 
+def test_get_event_status_completed():
     test_starttime_string = "2020-12-16T08:45:03Z"
     test_endtime_string = "2020-12-16T08:45:03Z"
     test_event_status = icone_translator.get_event_status(test_starttime_string, test_endtime_string)
     valid_event_status = "completed"
-
     assert test_event_status == valid_event_status
 
+    
 
+#--------------------------------------------------------------------------------Unit test for wzdx_creator function--------------------------------------------------------------------------------
 def test_wzdx_creator() :
-    wzdx_schema = json.loads(open('translator/sample files/validation_schema/wzdx_v3.0_feed.json').read())
-    icone_data =open('translator/sample files/Icone Data/incidents_extended.xml').read()
-
-
-    icone_obj = xmltodict.parse(icone_data)
-
-    # info={}
-    # info['feed_info_id'] = "feed_info_id"
-    #
-    # #### This information is required, might want to hardcode
-    # info['metadata'] = {}
-    # info['metadata']['wz_location_method'] = "channel-device-method"
-    # info['metadata']['lrs_type'] = "lrs_type"
-    # info['metadata']['location_verify_method'] = "location_verify_method"
-    # info['metadata']['datafeed_frequency_update'] = 86400
-    # info['metadata']['timestamp_metadata_update'] = "timestamp_metadata_update"
-    # info['metadata']['contact_name'] = "contact_name"
-    # info['metadata']['contact_email'] = "contact_email"
-    # info['metadata']['issuing_organization'] = "issuing_organization"
-    test_wzdx=icone_translator.wzdx_creator(icone_obj,icone_translator.initialize_info())
-
-    validate(instance=test_wzdx,schema=wzdx_schema)
-
-
+  icone_obj = {'incidents': {'incident': [{
+    '@id': 'U13631595_202012160845',
+    'updatetime': '2020-12-16T17:18:00Z',
+    'starttime': '2020-12-07T14:18:00Z',
+    'creationtime': '2020-12-13T14:18:00Z', 
+    'description': 'Road constructions are going on',
+    'location': {
+      'polyline': '34.8380671,-114.1450650,34.8380671,-114.1450650',
+      'street': 'I-70 N'
+    }
+  }]}}
+
+  #[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z 
```

Comment:
Can we remove?

Reply:
those are regular expression i used i did put it in comment. I just removed those .should be fixed